pkg/graph/schema.graphql

```diff
@@ -612,61 +612,216 @@ https://gqlgen.com/reference/changesets/
+  shouldAnyProviderExcludedFFSSystemsNote:        String!
+  changesMedicarePhysicianFeeSchedule:            Boolean!
+  changesMedicarePhysicianFeeScheduleNote:        String!
+  affectsMedicareSecondaryPayerClaims:            Boolean!
+  affectsMedicareSecondaryPayerClaimsExplanation: String!
+  affectsMedicareSecondaryPayerClaimsNote:        String!
+  payModelDifferentiation:                        String!
+
+  # Page 3
+  creatingDependenciesBetweenServices:     Boolean!
+  creatingDependenciesBetweenServicesNote: String!
+  needsClaimsDataCollection:               Boolean
+  needsClaimsDataCollectionNote:           String!
+  providingThirdPartyFile:                 Boolean!
+  isContractorAwareTestDataRequirements:   Boolean!
+
+  # Page 4
+  beneficiaryCostSharingLevelAndHandling:          String!
+  waiveBeneficiaryCostSharingForAnyServices:       Boolean!
+  waiveBeneficiaryCostSharingServiceSpecification: String!
+  waiverOnlyAppliesPartOfPayment:                  Boolean!
+  waiveBeneficiaryCostSharingNote:                 String!
+
+  # Page 5
+  nonClaimsPayments:                               [NonClaimsBasedPayType!]!
+  nonClaimsPaymentOtherDescription:                String!
+  paymentCalculationOwner:                         String!
+  numberPaymentsPerPayCycle:                       String!
+  numberPaymentsPerPayCycleNotes:                  String!
+  sharedSystemsInvolvedAdditionalClaimPayment:     String!
+  sharedSystemsInvolvedAdditionalClaimPaymentNote: String!
+  planningToUseInnovationPaymentContractor:        Boolean!
+  planningToUseInnovationPaymentContractorNote:    String!
+  fundingCenterDescription:                        String!
+
+  # Page 6
+  expectedCalculationComplexityLevel:                       ComplexityCalculationLevelType!
+  expectedCalculationComplexityLevelNote:                   String!
+  canParticipantsSelectBetweenPaymentMechanisms:            Boolean!
+  canParticipantsSelectBetweenPaymentMechanismsDescription: String!
+  canParticipantsSelectBetweenPaymentMechanismsNote:        String!
+  anticipatedPaymentFrequency:                              AnticipatedPaymentFrequencyType!
+  anticipatedPaymentFrequencyOtherDescription:              String!
+  anticipatedPaymentFrequencyNotes:                         String!
+
+  # Page 7
+  willRecoverPayments:                               Boolean!
+  willRecoverPaymentsNotes:                          String!
+  anticipateReconcilingPaymentsRetrospectively:      Boolean!
+  anticipateReconcilingPaymentsRetrospectivelyNotes: String!
+  paymentStartDate:                                  Time!
+  paymentStartDateNotes:                             String!
+
+  # Meta
+  createdBy: String!
+  createdDts: Time!
+  modifiedBy: String
+  modifiedDts: Time
+  status: TaskStatus!
+}
+
+input PlanPaymentsChanges @goModel(model: "map[string]interface{}") {
+  # Page 1
+  fundingSourceNote:                  String!
```

Comment:
With the exception of the arrays, these fields in the Changes object should not have an exclamation mark, as we don't require the full object to be filled out on submitting the update mutation. I got the below error


![image](https://user-images.githubusercontent.com/99294987/175362884-22292e3b-7de6-4be2-974c-2c3a52ceddcc.png)

Reply:
Specifically the fields _in_ the arrays should be non nullable. The arrays themselves can still be nullable!